ONSdigital/florence
# Feature/limit versions per collection

### What

Prevent publishing users adding more than one version to a collection. 

### How to review

- Upload a few versions
- Create a test collection, and add a version (should also add related dataset) to a collection
- Shouldn't let you edit another version or create a new version

### Who can review

Anyone
